robocorp-code/vscode-client/src/inspector.ts

```diff
@@ -0,0 +1,198 @@
+import * as path from 'path';
+import {commands, Progress, ProgressLocation, window} from 'vscode'
+import {ROBOCORP_GET_LANGUAGE_SERVER_PYTHON_INFO} from './robocorpCommands';
+import {getRccLocation} from './rcc'
+import {getExtensionRelativeFile, verifyFileExists} from './files'
+import {listAndAskRobotSelection} from './activities'
+import {getSelectedLocator, getSelectedRobot, LocatorEntry} from './viewsCommon';
+import {sleep, Timing} from './time'
+import {execFilePromise, ExecFileReturn} from './subprocess'
+import * as roboConfig from './robocorpSettings';
+import {OUTPUT_CHANNEL} from './channel';
+
+interface InterpreterInfo {
```

Comment:
Both declarations of `InterpreterInfo` in this module should probably be removed (this interface is already available in `extension.ts`, so if needed it can be imported from there).

Reply:
Removed